apps/arweave/src/ar_nonce_limiter.erl

```diff
@@ -546,7 +554,6 @@ handle_cast({validated_steps, Args}, State) ->
 					Session3#vdf_session.step_number),
 			Sessions2 = gb_sets:add_element({element(2, NextSessionKey),
 					element(1, NextSessionKey)}, Sessions),
-			ar_events:send(nonce_limiter, {validated_output, {SessionKey, Session2}}),
```

Comment:
Without this, we'll basically wait for the next VDF step that the VDF server computes, at which point it will try to push both the current and previous sessions, right? i.e. we don't need to immediately push the validated session since we can wait ~1 second for the next step to be computed and push the sessions then

Reply:
Yep, that's what I wanted to do for a start.

Reply:
Got it!